python/tvm/relay/quantize/quantize.py

```diff
@@ -74,7 +74,8 @@ class QConfig(NodeBase):
         "round_for_shift": True,
         "store_lowbit_output": True,
         "debug_enabled_ops": None,
-        "use_stop_fusion": True
+        "use_stop_fusion": True,
+        "quantize_op": [_op.get("nn.conv2d"), _op.get("nn.dense")]
```

Comment:
Remove this. If you want to partially enable some ops, it can be achieved by `debug_enabled_ops`.

Reply:
@ZihengJiang Done